### PR TITLE
[DE] Add `Help_centre/Beatmapping`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Beatmapping/de.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Ich bin ein osu!supporter, aber ich habe keine 20 ausstehenden Slots! {#upload-slots}
+
+**Die Zahl der verfügbaren Slots hängt von der Zahl der gerankten Beatmaps ab, die du hast. Für mehr Details, siehe [osu!supporter § Erhöhte Grenzwerte](/wiki/osu!supporter#erhöhte-grenzwerte).**
+
+## Wie kann ich meine Beatmap löschen? {#delete-beatmap}
+
+**Das [Support-Team](/wiki/People/The_Team/Account_support_team) nimmt keine Anfragen an, um hochgeladene Beatmaps manuell zu löschen.**
+
+Du kannst deine Beatmap selbst löschen, indem du wartest, dass sie auf dem Beatmap-Friedhof verfällt, was nach einem Monat (28 Tage) ohne Aktualisierungen passieren wird.
```
Comment (The-Last-Cookie) [inline wiki/Help_centre/Beatmapping/de.md:49]: `verfallen` is a very close translation to `to decay` but I think `landen` sounds more natural in this context. So after reconsideration, I suggest to use `dass sie auf dem Beatmap-Friedhof landet`.

Comment (FamousPig) [reply wiki/Help_centre/Beatmapping/de.md:49]: Sounds good!

**Review by FamousPig — COMMENTED** [review]
Everything apart from the above looks good!

File: wiki/Help_centre/Beatmapping/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Help_centre/Beatmapping/de.md:49]:
```suggestion
Du kannst deine Beatmap selbst löschen, indem du wartest, dass sie auf dem Beatmap-Friedhof landet, was nach einem Monat (28 Tage) ohne Aktualisierungen passieren wird.
```

**Review by respektive — APPROVED** [review]

**Pull request merged**